python/py-pythonz/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        berdario pythonz 1.11.4
+github.tarball_from releases
+name                py-${github.project}
+
+master_sites        ${github.homepage}/archive/
+distname            ${github.project}-${github.version}
+worksrcdir          ${github.project}-${github.project}-${github.version}
+
+platforms           darwin
+license             MIT
+
+python.versions     35 36 37
+
+maintainers         {gmail.com:rubendibattista @rubendibattista} openmantainer
+description         Python installation manager supporting CPython, Stackless, PyPy and Jython
+long_description    $description
+checksums           rmd160  fbf4687ea656bfbd85fc1a54caa2f7914d0bf616 \
+                    sha256  59d688ff0703cf2b62f3eb16344b944284b140cdb187f15c4a1d92ede2dcba60 \
+                    size    124201
+use_zip             yes
+
```

Comment:
please use the typical ```if {${subport} ne ${name}} {``` block for the installation. Since the package installs a ```console_script``` you'll actually need to declare a ```depends_lib``` on ```py-setuptools```. Additionally, it looks like you'll need ```resumable-urlretrieve``` for Python 3.5+; please check the setup.py to make sure you have all dependencies correct. Finally, please set ```livecheck.type none``` within this block. You can look at other Python ports for examples.